webpack/extend/CardItem.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable camelcase */
+
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Skeleton from 'react-loading-skeleton';
+import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  OkIcon,
+  ErrorCircleOIcon,
+} from '@patternfly/react-icons/dist/js/icons';
+import {
+  PropertiesSidePanel,
+  PropertyItem,
+} from '@patternfly/react-catalog-view-extension';
+
+import { get } from 'foremanReact/redux/API';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import CardItem from 'foremanReact/components/common/CardTemplate/CardItem';
+import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
+
+const ExtendedCardItem = ({ hostDetails: { name } }) => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (name) {
+      dispatch(
+        get({
+          key: 'HOST_DETAILS_JOBS',
+          url: `/api/job_invocations?search=host%3D${name}&per_page=3`,
+        })
+      );
+    }
+  }, [dispatch, name]);
+  const jobs = useSelector((state) =>
+    selectAPIResponse(state, 'HOST_DETAILS_JOBS')
+  );
+
+  const iconMarkup = (status) => {
+    if (status === 1) return <ErrorCircleOIcon style={{ color: 'red' }} />;
+    return <OkIcon style={{ color: 'green' }} />;
+  };
+
+  return (
+    <CardItem header="Recent Jobs">
+      <PropertiesSidePanel>
+        {jobs?.results?.map(
```

Comment:
how does the card look if we didn't find any job for the host? I think we should display some "No jobs for this host found" text :-)

Reply:
We have a pf4 empty state for this case :)
How about a general empty state for cards on the host details page?
I've opened an [issue](https://projects.theforeman.org/issues/31316) for this.